app/views/moderations/actions_panel.html.erb

```diff
@@ -86,45 +86,76 @@
+          data-article-id="<%= @moderatable.id %>"
+          data-group="experience_level">
+          <div class="level-rating-content">
+            <span class="level-rating-number">1</span>
+            <span class="level-rating-text">Beginner</span>
+          </div>
+          <%= inline_svg_tag("checkmark.svg", aria: true, class: "crayons-icon", title: "Checkmark") %>
+        </button>
+        <button
+          value="2"
+          name="rating_vote_2"
+          id="js__rating__vote__2"
+          class="level-rating-button <%= "selected" if @rating_vote&.rating == 2.0 %>"
+          data-user-id="<%= current_user.id %>"
+          data-article-id="<%= @moderatable.id %>"
+          data-group="experience_level">
+          <div class="level-rating-content">
+            <span class="level-rating-number">2</span>
+            <span class="level-rating-text">Novice</span>
+          </div>
+          <%= inline_svg_tag("checkmark.svg", aria: true, class: "crayons-icon", title: "Checkmark") %>
+        </button>
+        <button
+          value="3"
+          name="rating_vote_3"
+          id="js__rating__vote__3"
+          class="level-rating-button <%= "selected" if @rating_vote&.rating == 3.0 %>"
+          data-user-id="<%= current_user.id %>"
+          data-article-id="<%= @moderatable.id %>"
+          data-group="experience_level">
+          <div class="level-rating-content">
+            <span class="level-rating-number">3</span>
+            <span class="level-rating-text">Mid-level</span>
+          </div>
+          <%= inline_svg_tag("checkmark.svg", aria: true, class: "crayons-icon", title: "Checkmark") %>
+        </button>
+        <button
+          value="4"
+          name="rating_vote_4"
+          id="js__rating__vote__4"
+          class="level-rating-button <%= "selected" if @rating_vote&.rating == 4.0 %>"
+          data-user-id="<%= current_user.id %>"
+          data-article-id="<%= @moderatable.id %>"
+          data-group="experience_level">
+          <div class="level-rating-content">
+            <span class="level-rating-number">4</span>
+            <span class="level-rating-text">Advanced</span>
+          </div>
+          <%= inline_svg_tag("checkmark.svg", aria: true, class: "crayons-icon", title: "Checkmark") %>
+        </button>
+        <button
+          value="5"
+          name="rating_vote_5"
+          id="js__rating__vote__5"
+          class="level-rating-button <%= "selected" if @rating_vote&.rating == 5.0 %>"
+          data-user-id="<%= current_user.id %>"
+          data-article-id="<%= @moderatable.id %>"
+          data-group="experience_level">
+          <div class="level-rating-content">
+            <span class="level-rating-number">5</span>
+            <span class="level-rating-text">Expert</span>
+          </div>
+          <%= inline_svg_tag("checkmark.svg", aria: true, class: "crayons-icon", title: "Checkmark") %>
+        </button>
```

Comment:
I think this works, but IMO seems like an opportunity to refactor this here. Maybe an `each` loop and a hash containing the rating level texts? For example:
```
<% rating_hash = { "Expert" => 5, "Advanced" => 4 } # etc etc %>
<%= rating_hash.each do |rating_name, rating_level| %>
<button>
  # etc etc
  <span class="level-rating-number"><%= rating_level %></span>
  <span class="level-rating-text"><%= rating_name %></span>
</button>
<% end %>
```